--- a/metasploit-framework.gemspec
+++ b/metasploit-framework.gemspec
@@ -85,7 +85,7 @@
   # NTLM authentication
   spec.add_runtime_dependency 'rubyntlm'
   # Needed by for XML parsing
-  spec.add_runtime_dependency 'nokogiri'
+  spec.add_runtime_dependency 'nokogiri', '1.13.10'
   # Needed by db.rb and Msf::Exploit::Capture
   spec.add_runtime_dependency 'packetfu'
   # For sniffer and raw socket modules

